external/build/fugu-showcase.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * @fileoverview Fetches the Fugu Projects following the data process described
+ * in https://github.com/GoogleChromeLabs/fugu-showcase#data-flow.
+ */
```

Comment:
I am not a fan of this 😕 currently our `build-external` package script takes roughly 20 seconds. Running this new script alone took 6.3m. As the job is run async and end-users only load the readily prepared data that doesn't hurt too much, but I have a two other concerns

1) every potential deface (or similar) of a site would end up being screenshotted and displayed on ours
2) this script adds over 300 new files to our repository that would be overwritten once in a while, without the user really knowing why. I think the images should rather be served from the GCS uploader bucket.

Reply:
By looking at the job output, it also shows why this isn't ideal: 
![image](https://user-images.githubusercontent.com/12857772/211015948-42b43581-7c42-4d09-8e7a-245b2fb503a2.png) ![image](https://user-images.githubusercontent.com/12857772/211015980-5ad9cb4d-18cf-4c34-9f31-df9855f71893.png)

Reply:
Funny enough when you run the job you get surprising sound output out of nowhere as some of the apps are a11y focused 😄